.github/PULL_REQUEST_TEMPLATE_nolabels

```diff
@@ -0,0 +1,13 @@
+## ATTENTION - Release Note ack required!
+
+Update the below named `release-note-needed` block to one of:
+
+**\`\`\`release-note**
+**\`\`\`release-note-action-required**
```

Comment:
Adding new categories/labels is probably a separate item, but we can probably munge it in here.  New labels require changes across doc, mungers and release-time scrapers.